.github/workflows/publish.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
             type=sha,format=short,event=branch
             type=semver,pattern={{version}}
             type=raw,value=latest-ci,enable={{is_default_branch}}
-            type=raw,value=latest-ci_{{steps.current-time.outputs.unix_time}},enable={{is_default_branch}}
+            type=raw,value=latest-ci_${{steps.current-time.outputs.unix_time}},enable={{is_default_branch}}
```

Comment:
Does that fix need to apply to all the other uses of `{{}}` in these lines, too?

Reply:
tbh. I don't have an idea... I saw lines with and without $

Reply:
https://docs.github.com/en/actions/learn-github-actions/expressions

Reply:
Okay, well let's try this and see what it does